content/pages/disability-benefits/apply-for-benefits.md

```diff
@@ -197,9 +197,9 @@ We'll get your entire claim decision packet ready to be mailed.
 
 <li>
 
-**Claim Closed**
+**Claim Complete**
 
-We'll mail you a packet with a decision notice and award details. Please allow 5 to 7 business days for your packet to arrive before you contact a VA call center.
+We'll mail you a packet by U.S. mail that includes details of the decision on your claim. Please allow 7 to 10 business days for your packet to arrive before contacting a VA call center.
```

Comment:
I don't like the repetition of mail. How about "We'll send you a packet by U.S. mail"?